server/server.js

```diff
@@ -16,8 +16,30 @@ app.use(express.json());
 
 app.get('/', (req, res) => {
     res.json('nuevo comienzo')
-})
+});
+
+app.get('/users', async (req, res) => {
+    try {
+        const { rows: users } = await db.query('SELECT * FROM users');
+        res.send(users);
+    } catch (error) {
+        console.error('Error in Database Query:', error);
+        return res.status(400).json({ error });
+    }
+});
+
+app.get('/entries', async (req, res) => {
+    try {
+        const { rows: entries } = await db.query('SELECT * FROM entries');
+        res.send(entries);
+    } catch (error) {
+        console.error('Error in Database Query:', error);
+        return res.status(400).json({ error });
```

Comment:
same comment from ed above (https://github.com/Whitney-Rene/rose_aTC/pull/1/files#r1373426827)

Reply:
Thank you!